### DISCUSSION TITLE
Best way to store 2500000 record daily

### DISCUSSION THREAD

**Discussion by BigGitWorld** [isar/isar#1331] [Help]
Hello everyone.
I have an IoT system with 30 devices so that each device sends 55 byte of data (int, double and bool) every 1 second including temperature, humidity, alarms and etc.
I need to store these values to be able to do search/query on them to create reports, charts and etc.
Do you think what is the best database option for this scenario?
Kafka or Isar or anything else?

Thank you

**Comment by BigGitWorld** [comment]
I used isar successfully.

**Reply by ndelanou** [reply]
Thank you for the feedback 👍 
Did you notice any drawbacks regarding performances when scaling to millions ?
